assets/js/modules/adsense/datastore/settings.js

```diff
@@ -219,6 +220,9 @@ export const controls = {
 		}
 
 		await API.invalidateCache( 'modules', 'adsense' );
+		// TODO: Remove once legacy dataAPI is no longer used.
+		dataAPI.invalidateCacheGroup( TYPE_MODULES, 'adsense' );
```

Comment:
Just curious whether importing the object significantly increases the bundle size for these module assets? If so, adding an extra export for just the function would reduce that overhead?

Reply:
Good question 😄 

Apparently it has no effect on the AdSense module bundle because it was already in there to be used by `isAdsenseConnectedAnalytics`.
https://github.com/google/site-kit-wp/blob/3a21b1a16689d3e147b7a44bd0d07df4de604b17/assets/js/modules/adsense/util/index.js#L29

For Analytics, it increased the production bundle size from `28.72KB` to `34.4KB`. I would say that is probably not worth splitting out another export for, although I would imagine the amount of code we actually need would probably leave this under 30KB, but that's just a guess.

Reply:
Let's just keep it as it is for now.